cccs-build/superset-base/azure-pipelines.yml

```diff
@@ -12,8 +12,7 @@ variables:
   buildTimestamp: $[format('{0:yyyyMMddHHmmss}', pipeline.startTime)]
   DOCKER_BUILDKIT: 1
 
-pool:
-  vmImage: ubuntu-latest
+pool: "cauldron-vmss"
```

Comment:
I'm a big fan of omitting the quotes whenever possible in YAML, so I think we should remove them (from this line, as well as the two `command` lines in this file, as well as the `pool` line in the other file. Otherwise, I'm good with this! :)